_security/configuration/configuration.md

```diff
@@ -306,8 +307,9 @@ Name | Description
 `jwt_url_parameter` | If the token is not transmitted in the HTTP header, but as an URL parameter, define the name of this parameter here.
 `subject_key` | The key in the JSON payload that stores the user name. If not set, the [subject](https://tools.ietf.org/html/rfc7519#section-4.1.2) registered claim is used.
 `roles_key` | The key in the JSON payload that stores the user's roles. The value of this key must be a comma-separated list of roles.
+`jwt_clock_skew_tolerance_seconds` |  Sets a window of time in seconds to prevent authentication failures due to a misalignment between clock times for the JWT authentication server and OpenSearch node. Security sets 30 seconds as the default. Use this setting to apply a custom value.
```

Comment:
```suggestion
`jwt_clock_skew_tolerance_seconds` |  Sets a window of time, in seconds, to prevent authentication failures due to a misalignment between clock times for the JWT authentication server and OpenSearch node. Security sets 30 seconds as the default. Use this setting to apply a custom value.
```